<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateExecutableCommandWidget</class>
 <widget class="QWidget" name="CreateExecutableCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCommandLinkButton" name="cbImportProgram">
     <property name="text">
      <string>Select program</string>
     </property>
     <property name="description">
      <string>Select an installed program from a list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="text">
      <string>Manual</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="description">
      <string>Configure the needed parameters manually</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Executable:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KUrlRequester" name="urExecutable">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Working Directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlRequester" name="urWorkingDirectory">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>99</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>commandLinkButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>urExecutable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_12</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_13</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>urWorkingDirectory</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
